packages/SwingSet/src/kernel/liveSlots.js

```diff
@@ -764,13 +765,37 @@ function build(
     const rootObject = buildRootObject(harden(vpow), harden(vatParameters));
     assert.equal(passStyleOf(rootObject), REMOTE_STYLE);
 
-    const rootSlot = makeVatSlot('object', true, 0n);
+    const rootSlot = makeVatSlot('object', true, BigInt(0));
```

Comment:
We can't have nice things yet? It's not clear why not.

Reply:
In this case it was my local editor (`js2-mode` in emacs) that complained, and it was an easy fix for a small papercut. It doesn't complain nearly as badly as `const f = (arg1, arg2) => { return blah; }`, which complains about a "missing formal parameter", or `foo: (arg) => { blah }`, which seems to break the parser entirely (as well as my eyeballs).

I'll look around to see if that mode has been updated to know about modern syntax changes.